translations/inferno_devilish_plan.tex/cs.po

```diff
@@ -540,6 +659,12 @@ msgid ""
 "\\vspace*{\\fill}\n"
 "\n"
 msgstr ""
+"\\vspace*{\\fill}\n"
+"\\begin{center}\n"
+"\\includegraphics[width=0.3\\paperwidth, keepaspectratio]{\\art/slayer.png}\n"
```

Comment:
Why is it smaller?

Reply:
It didn't look good at full scale - too close to the rest of the text

<img width="1001" alt="Screenshot 2024-08-02 at 22 56 36" src="https://github.com/user-attachments/assets/bc6a9c45-600b-4d94-8106-32db2d6e1718">